salt/vault/secret_backends.sls

```diff
@@ -15,6 +17,28 @@ enable_mitx_aws_secret_backend:
     - lease_max: {{ SIX_MONTHS }}
     - lease_default: {{ SIX_MONTHS }}
 
+enable_pki_intermediate_backend:
+  vault.secret_backend_enabled:
+    - backend_type: pki
+    - mount_point: pki_int
+    - description: Backend to create self-signed certificates
+    - ttl_default: {{ pki_ttl }}
+    - connection_config:
+        issuing_certificates: 'https://vault.service.consul:8200/vi/pki_int/ca'
+        crl_distribution_points: 'https://vault.service.consul:8200/v1/pki_int/crl'
+
+{% for environment in env_settings.environments %}
+enable_pki_intermediate_{{ environment }}_backend:
+  vault.secret_backend_enabled:
+    - backend_type: pki
+    - mount_point: pki_int_{{ environment }}
```

Comment:
Same as above with using `-`, also, maybe use a name like `pki-{{ environment }}-ca`? I'm happy to stick with `pki-{{environment }}-intermediate-ca` as well